bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/airbase/AirbaseModelInfo.java

```diff
@@ -35,7 +35,7 @@ public class AirbaseModelInfo {
     public String ret = "";
     public int zonespresent;
     public int commonzone;
-    public int frate_steps; // fan rate steps
+    public int fRateSteps;
```

Comment:
Keep the comment, please.

Reply:
I thought the var name would be self-explanatory, placed the comment back in.

Reply:
the `f` isn't for `fan`. All the params start with `f_xxxx`. So we're left with "rate steps". Hence the comment.